streams/src/test/java/org/apache/kafka/streams/integration/utils/KafkaEmbedded.java

```diff
@@ -42,7 +28,18 @@
 import kafka.utils.TestUtils;
 import kafka.utils.ZKStringSerializer$;
 import kafka.utils.ZkUtils;
+import org.I0Itec.zkclient.ZkClient;
```

Comment:
Is this imports re-organization intentional? Basically we want to group different categories of imports together.